### ISSUE TITLE
Temp table droping issue on Npgsql (PostgreSQL) context on BulkInsertOrUpdateAsync operation 

### ISSUE THREAD

**Issue by Kolyan4ik95** [borisdj/EFCore.BulkExtensions#858]
Npgsql.EntityFrameworkCore.PostgreSQL 6.0.3 
Microsoft.EntityFrameworkCore 6.0.2
EFCore.BulkExtensions 6.4.0

I have method running many times on single transaction: 
``` csharp
private static async Task<int> SaveItemsAsync(DbContext context, IList<TModel> entities, 
                int batchSize = 10000, CancellationToken token = default)
{
    var bulkConfig = new BulkConfig
    {
        SetOutputIdentity = false,
        TrackingEntities = false,
        CalculateStats = false,
        BatchSize = batchSize,
        EnableStreaming = true,
        UseTempDB = true, //here is trouble 
    };
    await context.BulkInsertOrUpdateAsync(entities, bulkConfig, cancellationToken: token);    
    return entities.Count;
}
```
My datacontext builder  : 
``` csharp
var builder = new NpgsqlConnectionStringBuilder(connectionString);
optionsBuilder
    .UseNpgsql(builder.ConnectionString, o => o.SetPostgresVersion(9, 6));
```

After transaction commit i expect the temporary tables was dropped but all of them are exist. 

Current behavior create SQL script:
``` sql 
CREATE TABLE "#TModelTemp242b578a"
                AS TABLE "TModel"
                WITH NO DATA
```
but IMHO it should be 
``` sql 
CREATE TEMP /*TEMPORARY*/ TABLE "#TModelTemp242b578a"
                AS TABLE "TModel"
                WITH NO DATA
```

If **UseTempDB**  set to **false** it was run as expected, but each of tables was dropped manually by script

**Comment by borisdj** [comment]
Prefix '#' is SqlServer syntax for TempDp table, but as you stated PG is using another keyword TEMP or TEMPORARY.
Will see to fix it with next version.

**Comment by borisdj** [comment]
Fixed with v6.5.3
